src/main/java/net/sf/jsqlparser/expression/DateTimeLiteralExpression.java

```diff
@@ -53,6 +53,20 @@ public DateTimeLiteralExpression withType(DateTime type) {
     }
 
     public enum DateTime {
-        DATE, TIME, TIMESTAMP, TIMESTAMPTZ;
+        DATE("DATE"), TIME("TIME"), TIMESTAMP("TIMESTAMP"), TIMESTAMPTZ("TIMESTAMPTZ");
```

Comment:
The purpose is `Case Insensitive Assignment of ENUMs` to `Token.image` (without writing `toUpperCase` everytime).
There was actually an issue reported about that and so I did this consistently for all Enums (since I want to keep the Grammar as lean as possible).

Applies for all Enums mentioned below.

Reply:
You are right, it is not necessary for the case-insensitive assignment and I have reworked it.